release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -2154,6 +2154,8 @@ To make the required change, follow the instructions in link:https://access.redh
 
 * In hosted control planes for {product-title}, on the bare metal and {VirtProductName} platforms, the auto-repair function is disabled. (link:https://issues.redhat.com/browse/OCPBUGS-20028[*OCPBUGS-20028*])
 
+* For oc-mirror mirror action, if the source catalog bundles an architecture specific opm, you should execute the mirroring from that architecture. For instance, if you mirror a ppc64le catalog, you should run from a power machine. If you run from another architecture. (link:https://issues.redhat.com/browse/OCPBUGS-22264[*OCPBUGS-22264*])
```

Comment:
Since the source catalog bundles an architecture specific opm binary, you should run the mirroring from that architecture.  For instance if you are mirroring a ppc64le catalog, you should run oc-mirror from a system that runs on the ppc64le architecture.  (link:https://issues.redhat.com/browse/OCPBUGS-22264[*OCPBUGS-22264*])

Reply:
@manojnkumar I have updated the PR as per above. Thanks.